specs/server/test_tmdb.js

```diff
@@ -0,0 +1,195 @@
+const { expect } = require('chai');
+// const request = require('supertest');
+var { searchMoviesByName, fetchMovieById, fetchImageById } = require('../../utils/tmdb.js');
+var api_key = process.env.API_KEY || null;
+var id = 10195;
+
+describe ('test tmdb server functions', () =>{
+  describe('searchMoviesByName', () => {
+    var data;
+    before (done => {
+      searchMoviesByName('thor')
+      .then(res => {
+        data = res;
+        done();
+      })
+    })
+    it ('returned information should be a object', () =>{
+      console.log( Object.keys(data))
+      expect(data).to.be.a('object');
+    })
+
+    it ('should expect data to contain: page, total_results, total_pages, and results', () =>{
+      expect(data.page).to.exist
+      expect(data.total_results).to.exist
+      expect(data.total_pages).to.exist
+      expect(data.results).to.exist
+    })
+    it ('results should contain a array of movies', () =>{
+    //   { vote_count : 7176,
```

Comment:
I think this should be a test file in the repo and not comment text here.

Reply:
yea i agree, i was trying to avoid creating too many files